cli/commands/data_backup.go

```diff
@@ -37,8 +38,12 @@ func (cmd *DataBackup) Commands() []cli.Command {
 }
 
 func (cmd *DataBackup) Run(c *cli.Context) error {
+	if runtime.GOOS == "linux" {
```

Comment:
Given the need to add runtime and check on Linux everywhere, I wonder if we shouldn't add a util/feature_detection.go now and define util.CanRunNative() as a check for linux.

Reply:
`util.NativeDockerSupport()` ?

Reply:
In anticipation of other things to detect, `util.SupportsNativeDocker()`.

`util.SupportsFileSync()` would currently exclude windows without bash right?